boards/arm/nucleo_l4r5zi/nucleo_l4r5zi.dts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2017 Pushpal Sidhu
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/stm32l4r5.dtsi>
+
+/ {
+	model = "STMicroelectronics STM32L4R5ZI-NUCLEO board";
+	compatible = "st,stm32l4r5zi-nucleo", "st,stm32l4r5";
+
+	chosen {
+		zephyr,console = &usart2;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+};
+
+&usart2 {
```

Comment:
This would be usart1.
Then do you want to activate USART 1?
This is not enabled in _defconfig file

Reply:
Oh shoot, I might have missed that, thanks.